Task_12/truck_delivery_rayus007/util/error_handler.py

```diff
@@ -0,0 +1,12 @@
+from truck_delivery_rayus007.util.custom_exceptions import CustomException
```

Comment:
this was just an example, you can delete it as it's not working and it shouldn't be here

Reply:
removed.